go.mod

```diff
@@ -150,13 +150,13 @@ require (
 	go.uber.org/fx v1.21.1
 	go.uber.org/multierr v1.11.0
 	go.uber.org/zap v1.27.0
-	golang.org/x/crypto v0.23.0
+	golang.org/x/crypto v0.24.0
 	golang.org/x/net v0.25.0
 	golang.org/x/sync v0.7.0
-	golang.org/x/sys v0.20.0
-	golang.org/x/term v0.20.0
+	golang.org/x/sys v0.21.0
+	golang.org/x/term v0.21.0
 	golang.org/x/time v0.5.0
-	golang.org/x/tools v0.21.0
+	golang.org/x/tools v0.21.1-0.20240508182429-e35e4ccd0d2d
```

Comment:
why are these all updated in this PR? updating `github.com/jackc/pgx/v5` shouldn't have bumped any other dependency

Reply:
> why are these all updated in this PR? updating `github.com/jackc/pgx/v5` shouldn't have bumped any other dependency

sorry, I used the `go get -u `